src/app/modules/convai-checker/convai-checker.component.spec.ts

```diff
@@ -1369,142 +1381,19 @@ describe('Convai checker test', () => {
-      expectedFeedbackTextVisibilitiesAfterLoading,
-      widgetId);
-  });
-
-  it('Test loading icon visibility, alwaysHideLoadingIcon = true, min threshold > 0',
-     async() => {
-    const fixture = TestBed.createComponent(test_components.ConvaiCheckerCustomDemoSettingsComponent);
-    fixture.detectChanges();
-    // Show feedback above a minimum threshold, but never show the loading icon.
-    const demoSettings = getCopyOfDefaultDemoSettings();
-    demoSettings.scoreThresholds = [0.4, 0.6, 0.8];
-    demoSettings.alwaysHideLoadingIcon = true;
-    demoSettings.hideLoadingIconAfterLoad = false;
-    demoSettings.hideLoadingIconForScoresBelowMinThreshold = false;
-    const mockResponseScores = [0.6, 0, 0.9];
-    const expectedWidgetVisibilitiesBeforeLoading = [false, false, false];
-    const expectedWidgetVisibilitiesWhileLoading = [false, false, false];
-    const expectedWidgetVisibilitiesAfterLoading = [false, false, false];
-    const expectedFeedbackTextVisibilitiesAfterLoading = [true, false, true];
-
-    await verifyWidgetVisibilityForDemoSettings(
-      fixture,
-      httpMock,
-      demoSettings,
-      mockResponseScores,
-      expectedWidgetVisibilitiesBeforeLoading,
-      expectedWidgetVisibilitiesWhileLoading,
-      expectedWidgetVisibilitiesAfterLoading,
-      expectedFeedbackTextVisibilitiesAfterLoading);
-  });
-
-  it('Test loading icon visibility, alwaysHideLoadingIcon = true, min threshold > 0, emoji icon',
-     async() => {
-    const fixture = TestBed.createComponent(test_components.ConvaiCheckerCustomDemoSettingsComponent);
-    fixture.detectChanges();
-    // Show feedback above a minimum threshold, but never show the loading icon.
-    const demoSettings = getCopyOfDefaultDemoSettings();
-    demoSettings.scoreThresholds = [0.4, 0.6, 0.8];
-    demoSettings.alwaysHideLoadingIcon = true;
-    demoSettings.hideLoadingIconAfterLoad = false;
-    demoSettings.hideLoadingIconForScoresBelowMinThreshold = false;
-    demoSettings.loadingIconStyle = LoadingIconStyle.EMOJI;
-    const mockResponseScores = [0.6, 0, 0.9];
-    const expectedWidgetVisibilitiesBeforeLoading = [false, false, false];
-    const expectedWidgetVisibilitiesWhileLoading = [false, false, false];
-    const expectedWidgetVisibilitiesAfterLoading = [false, false, false];
-    const expectedFeedbackTextVisibilitiesAfterLoading = [true, false, true];
-    const widgetId = 'emojiStatusWidget';
-
-    await verifyWidgetVisibilityForDemoSettings(
-      fixture,
-      httpMock,
-      demoSettings,
-      mockResponseScores,
-      expectedWidgetVisibilitiesBeforeLoading,
-      expectedWidgetVisibilitiesWhileLoading,
-      expectedWidgetVisibilitiesAfterLoading,
-      expectedFeedbackTextVisibilitiesAfterLoading,
-      widgetId);
-  });
-
-  it('Test loading icon visibility, hideLoadingIconForScoresBelowMinThreshold = true, '
-     + ' min threshold = 0',
+  // TODO: Add a test for hiding feedback for both low and neutral scores.
```

Comment:
Is this TODO for the tests below? I noticed a couple of them don't have any asserts/expects so I wasn't sure if this applied to those.

Reply:
Done, thanks for catching! These tests actually call a function that has asserts/expects, but there were two tests I had meant to add which I forgot.